_functions/synth_parameters/delay.md

```diff
@@ -2,4 +2,5 @@
 title: delay
 category: synth_parameters
 ---
-a pattern of numbers from 0 to 1. Sets the level of the delay signal.
+a pattern of numbers that set the initial level of the delay signal.  I.e. a value of one means the first echo will be as
```

Comment:
ah sure, sorry: of course `delayAmp` works that way. I was talking about `delaytime`, but actually I just saw that we have been clipping it anyhow > 0.